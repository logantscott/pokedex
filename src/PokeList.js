import React, { Component } from 'react';
import PokeItem from './PokeItem.js';
import shortid from 'shortid';

export default class PokeList extends Component {

  render() {
    return (
      <ul className="grid-container">
          {this.props.pokedex.map(pokemon => (
          <PokeItem pokemon={pokemon} key={shortid.generate()} />
          ))}
      </ul>
    );
  }
}