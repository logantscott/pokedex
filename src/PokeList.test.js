import React from "react";
import PokeList from "./PokeList.js";
import renderer from "react-test-renderer";

test('renders PokeList.js correctly', () => {
  const tree = renderer
    .create(<PokeList pokeData={[]} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});