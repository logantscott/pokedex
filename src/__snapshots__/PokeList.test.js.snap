// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PokeList.js correctly 1`] = `
<ul
  className="grid-container"
/>
`;
